internal/common/function/function_test.go

```diff
@@ -196,7 +198,10 @@ func TestGetBackupSpecFromNonAdminBackup(t *testing.T) {
 
 	assert.NoError(t, err)
 	assert.NotNil(t, backupSpec)
-	assert.Equal(t, backupSpecInput, backupSpec)
+	assert.Equal(t, []string{"namespace1"}, backupSpec.IncludedNamespaces) // Check that only the namespace from NonAdminBackup is included
+	assert.Equal(t, backupSpecInput.ExcludedNamespaces, backupSpec.ExcludedNamespaces)
+	assert.Equal(t, backupSpecInput.StorageLocation, backupSpec.StorageLocation)
+	assert.Equal(t, backupSpecInput.VolumeSnapshotLocations, backupSpec.VolumeSnapshotLocations)
 }
```

Comment:
add test for new error :question:

Reply:
ah right there is missing one. will do

Reply:
Added 2 new test cases to cover new error.